var/spack/repos/builtin/packages/py-uhi/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyUhi(PythonPackage):
+    """Unified Histogram Interface:
+       tools to help library authors work with histograms"""
+
+    homepage = "https://github.com/Scikit-HEP/uhi"
+    pypi     = "uhi/uhi-0.3.0.tar.gz"
+
+    version('0.3.0', sha256='3f441bfa89fae11aa762ae1ef1b1b454362d228e9084477773ffb82d6e9f5d2c')
+
+    depends_on('python@3.6:', type=('build', 'run'))
```

Comment:
Need build dep on `py-poetry-core@1:` once #27798 is merged. You can just comment it out for now, I don't think the package exists in develop yet. Until then, a build dep on `py-setuptools` is needed.